pkg/snmp/traps/oid_resolver.go

```diff
@@ -24,6 +24,15 @@ import (
 
 const ddTrapDBFileNamePrefix string = "dd_traps_db"
 
+var nodesOIDThatShouldNeverMatch = []string{
+	"1.3.6.1.4",
+	"1.3.6.1",
+	"1.3.6",
+	"1.3.6",
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏It can help readers to have the OID name as comment here, and possibly short desc why they should never match.